unittests/test-contracts/payloadless/payloadless.cpp

```diff
@@ -5,3 +5,46 @@ using namespace eosio;
 void payloadless::doit() {
    print("Im a payloadless action");
 }
+
+constexpr size_t cpu_prime_max = 15375u;
+
+bool is_prime(int p) {
+   if (p == 2) {
+      return true;
+   } else if (p <= 1 || p % 2 == 0) {
+      return false;
+   }
+
+   bool prime = true;
+   const int to = sqrt(p);
+   int i;
+   for (i = 3; i <= to; i += 2) {
+      if (!((prime = bool(p)) % i)) break;
+   }
+   return prime;
```

Comment:
As `prime` is initialized to true, would the function always returns true? On the other hand, this function only concerns about `slowness`.

Reply:
Oh I missed the assignment `prime = bool(p)`.

Reply:
I agree, this just always returns true. I'll fix it.